boa/interpret.py

```diff
@@ -181,21 +181,39 @@ def loads(
     name=None,
     filename=None,
     compiler_args=None,
+    verify: bool = False,
+    explorer: Optional[str] = None,
     **kwargs,
 ):
     d = loads_partial(source_code, name, filename=filename, compiler_args=compiler_args)
     if as_blueprint:
         return d.deploy_as_blueprint(**kwargs)
     else:
         return d.deploy(*args, **kwargs)
+        if verify:
```

Comment:
this code is repeated, why not just pass the arguments directly to deploy without (even just via the kwargs)